docs/logs/getting-started/README.md

```diff
@@ -62,11 +62,17 @@ instance, which is then used to do the logging. The log is sent to the
 `ConsoleExporter`. `ConsoleExporter` simply displays it on the console.
 
 Adding logging providers, obtaining `ILogger` instance, etc. could be done
-differently based on the application type. For example, [Logging in ASP.NET
-Core](https://docs.microsoft.com/aspnet/core/fundamentals/logging#logging-providers)
-shows how to do logging in ASP.NET Core, which is also demonstrated in the
-[OpenTelemetry Example ASP.NET Core
-application](../../../examples/AspNetCore/Program.cs)
+differently based on the application type depending on if the application type
+is already setting up an `ILogger` based logging pipeline.
+
+> **Note**
+> Application types like [ASP.NET Core](https://learn.microsoft.com/aspnet/core)
```

Comment:
It sounds like "application types set up the pipeline" while the actual intention is "applications set up the pipeline".

Consider this: "Certain types of applications (e.g. ASP.NET Core and Worker) have logging pipeline set up by default"